packages/frontend/src/components/SavedDashboards/CreateSavedDashboardModal.tsx

```diff
@@ -1,36 +1,73 @@
-import { CreateDashboard, Dashboard, DashboardBasicDetails } from 'common';
-import React, { FC } from 'react';
+import {
+    Button,
+    Classes,
+    Dialog,
+    FormGroup,
+    InputGroup,
+    Intent,
+} from '@blueprintjs/core';
+import { Dashboard } from 'common';
+import { FC, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { useCreateMutation } from '../../hooks/dashboard/useDashboard';
-import { ActionModalProps } from '../common/modal/ActionModal';
-import CreateActionModal from '../common/modal/CreateActionModal';
 
 interface CreateSavedDashboardModalProps {
     isOpen: boolean;
-    ModalContent: (
-        props: Pick<
-            ActionModalProps<DashboardBasicDetails>,
-            'useActionModalState' | 'isDisabled'
-        >,
-    ) => JSX.Element;
+
     tiles?: Dashboard['tiles'];
     showRedirectButton?: boolean;
     onClose?: () => void;
 }
 
-const CreateSavedDashboardModal: FC<CreateSavedDashboardModalProps> = (
-    props,
-) => {
+const CreateSavedDashboardModal: FC<CreateSavedDashboardModalProps> = ({
+    isOpen,
+    tiles,
+    showRedirectButton,
+    onClose,
+}) => {
     const { projectUuid } = useParams<{ projectUuid: string }>();
-    const { tiles, showRedirectButton } = props;
     const useCreate = useCreateMutation(projectUuid, showRedirectButton);
+    const { mutate, isLoading: isCreating } = useCreate;
+    const [name, setName] = useState<string>('');
 
     return (
-        <CreateActionModal
-            useCreate={useCreate}
-            savedData={{ tiles: tiles || [] } as Partial<CreateDashboard>}
-            {...props}
-        />
+        <Dialog isOpen={isOpen} onClose={onClose} lazy title="Create dashboard">
+            <form>
+                <div className={Classes.DIALOG_BODY}>
+                    <FormGroup
+                        label="Name"
+                        labelFor="chart-name"
+                        style={{ fontWeight: 'bold' }}
```

Comment:
Looks great, just if we could move this to its own styled component, for consistency.